.travis.yml

```diff
@@ -19,8 +19,11 @@ env:
   # Ensure that we build without warnings.
   - CARGO_FLAGS="--features 'strict'"
 
+before_script: (cargo install rustfmt || true)
```

Comment:
This will actually succeed if `rustfmt` can't be installed:

```
$ cargo install nosuchpackage || true
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: could not find `nosuchpackage` in `registry https://github.com/rust-lang/crates.io-index`
$ echo $?
0
```

However, we depend on it being installed when we call `cargo fmt` lower down. If I've understood correctly, at least.